integration/data/common.py

```diff
@@ -151,6 +152,12 @@ def restore_with_no_frontend(backup, grpc_c):
     assert v.frontendState == "up"
 
 
+def snapshot_revert_with_no_frontend(name):
```

Comment:
This is in fact `snapshot_revert_with_frontend`.

Reply:
The function disables the engine frontend before running snapshot_revert right?
Are you implying me to change it to `snpashot_revert_with_frontend_disable` ?

Reply:
I meant, this function disables the frontend, but it runs with frontend enabled. From the caller point of view, it's running the function when the frontend is enabled. The function knows what to do when the frontend is enabled (which is to disable it first). How to prepare the revert is not the concern of the caller.

Reply:
Agreed, will change this to `snapshot_revert_with_frontend`